src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py

```diff
@@ -223,10 +295,9 @@ def as_constant(self):
         if self.pt_module.op == 'get_attr':
             # Extract Constant from FX module field
             ret = fetch_attr(self.fx_gm, self.pt_module.target)
-            ov_const = op.Constant(ret.numpy(), shared_memory=True)
+            ov_const = op.Constant(ret.numpy(force=True), shared_memory=True)
```

Comment:
should be try except for force? I remember some issues when tensor does not have such attribute?

Reply:
Older versions of torch doesn't have `force`. So try-except doesn't makes sense for torchdynamo, since it wasn't available in those older versions of torch